tests/test_utils.sh

```diff
@@ -847,6 +847,16 @@ function test_grep_fields {
     rm -f "$filename"
 }
 
+function test_substitute {
+    local sub=(
```

Comment:
This breaks on Zsh 5.0:
```suggestion
    local sub
    sub=(
```